src/gui.rs

```diff
@@ -418,24 +146,14 @@ pub enum PrimitiveError {
 }
 
 impl Gui {
-    /// Creates a new [3D graphics primitive](Primitive3) from raw components.
-    pub fn make_primitive3(
-        &mut self,
-        vertices: &[Vertex3],
-        indices: &[Index],
-        texture: Rc<Texture>,
-    ) -> Result<Primitive3, PrimitiveError> {
-        self.backend.make_primitive3(vertices, indices, texture)
-    }
-
-    /// Creates a new [2D graphics primitive](Primitive2) from raw components.
-    pub fn make_primitive2(
+    /// Creates a new [3D graphics primitive](Primitive) from raw components.
```

Comment:
Maybe `[3D graphics primitive]` is not actual now because now this function responsible to render 2D graphics too.

Reply:
Reasonable. I'll address this when I introduce the primitive building API.